portality/migrate/autocomplete/autocomplete.json

```diff
@@ -0,0 +1,7 @@
+{
+	"types": [
+		{
+			"type": "journal"
```

Comment:
Don't applications (type suggestion) also have these auto-complete dropdowns and therefore require a self._generate_autocompletes() to be called on them as well?

Bit unusual to comment on a merged PR - just want to make sure we've done everything consistently.